config/sample-bigtreetech-hermit-crab-canbus.cfg

```diff
@@ -0,0 +1,128 @@
+# This file contains common pin mappings for the BIGTREETECH HermitCrab
+# Canbus board. To use this config, the firmware should be compiled for the
+# STM32F072 with "8 MHz crystal" and USB (on PA11/PA12) or CAN bus (on PB8/PB9).
+# The "HermitCrab" micro-controller will be used to control the components on the nozzle.
+
+[mcu HermitCrab]
+#canbus_uuid: 0e0d81e4210c
+#serial: /dev/serial/by-id/usb-Klipper_Klipper_firmware_12345-if00
+
+[adxl345]
+cs_pin: HermitCrab: PB12
+spi_software_sclk_pin: HermitCrab: PB13
```

Comment:
Any reason this is configured as a software SPI instead of a hardware SPI as
```
cs_pin: HermitCrab: PB12
spi_bus: spi2
```
?

Reply:
Modified to hardware SPI